source/_posts/2018-11-13-whining-vs-venting.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Whining vs Venting"
+date: 2018-11-13 08:14:11 -0700
+comments: false
+categories: management
+---
+It was time for my one-on-one with Ron and I could tell it would be a doozy. He slumped into a chair and before I could even start with a basic “how’s it going?” he launched into a tirade about Sales and how they were screwing us over *again* and that he had to pick up the extra load. This is a not exactly a unique case - there’s a natural tension between promise and delivery and there’s more than a kernel of truth to it. Ron was really steamed this time though. It could be he needed to vent, but maybe he was just whining.
+
+I don’t like whining. I have two kids who are approaching teenage-dom and I have enough of it in my life already, thank you very much. Whining from adults is especially grating - they should know better, but it still happens.
+
+I view whining as subtly but distinctly different than venting. And what is that difference you ask? Whining focuses on “woe is me!” and doesn’t lend itself to finding solutions. A whiner wants everything and everyone around them to change to accommodate themselves and their needs. Further, whining tends to beget more whining, whereas when a good old fashion vent is complete, progress can be made. Venting can be healthy.
+
+I actually kinda like hearing a good [vent](http://randsinrepose.com/archives/the-update-the-vent-and-the-disaster/) session. If someone is venting to me directly, then it means they have some level of [trust](http://blog.swilliams.me/words/2018/11/01/building-trust-on-teams/) with me. They’re opening up and spilling out a little bit of themselves. This means you need to be wearing your listening hat. Vents can get emotional - this is good, it means the venter is invested deeply in something and that they care. So, listen! With empathy!
```

Comment:
`Vents can get emotional - this is good, it means` -> `Vents can get emotional&mdash;this is good. It means...`